### ISSUE TITLE
Add option for ragged columns in LaTeX

### ISSUE THREAD

**Issue by morberg** [jdutant/columns#3]
I'm using your script to generate a PDF of a recipe collection in markdown. Ingredients are in two columns and the cooking instructions follows in one column layout.

Sometimes the list of ingredients does not fit evenly in two columns and LaTeX fills up with vertical space (too much white space in right column):
<img width="574" alt="image" src="https://user-images.githubusercontent.com/956034/109474335-b36d1e80-7a74-11eb-96f8-e035422d47e4.png">

By inserting
```latex
\vspace*{\fill}
```

before `\end{multicols}` I can get my desired result (see [this SO thread](https://tex.stackexchange.com/questions/131885/vfill-and-vertical-space#131889)):

<img width="618" alt="image" src="https://user-images.githubusercontent.com/956034/109474086-68eba200-7a74-11eb-9b12-3dbe4b8e5025.png">

It would be nice to have an option to not layout column text in the entire column height. This would insert `\vspace*{\vfill}` before `\columnbreak` (for left column) and `\end{multicols}` (for right column).

**Comment by morberg** [comment]
Here is an example of what happens when inserting `\vspace*{vfill}` before a `\columnbreak` (wanted result):
<img width="699" alt="image" src="https://user-images.githubusercontent.com/956034/109477270-20ce7e80-7a78-11eb-98fc-a949a61b645a.png">
And the current (undesired) output:
<img width="670" alt="image" src="https://user-images.githubusercontent.com/956034/109477422-4a87a580-7a78-11eb-8963-15eac5dc0113.png">

**Comment by morberg** [comment]
The correct way to solve it is probably to use `\raggedcolumns` instead. Create a file `format.tex` that contains:

```latex
\raggedcolumns
```

And then use
```bash
pandoc --lua-filter=columns.lua --include-before-body=format.tex file.md -o file.pdf
```
and Bob's your uncle.

**Comment by jdutant** [comment]
Thanks for your input and sorting this one. I was thinking of adding filter options, one could be to get ragged columns. Something like adding:

```yaml
columns-filter:
   ragged-columns: yes
```

in the file metadata. I'll keep this open as an enhancement.

**Comment by jdutant** [comment]
This is implemented with 93b66967a9eb8e325f902b2170c30c373ff35c84. See README.md for usage.
